subgraph.template.yaml

```diff
@@ -112,7 +93,7 @@ templates:
         - event: FPMMSell(indexed address,uint256,uint256,indexed uint256,uint256)
           handler: handleSell
         - event: Transfer(indexed address,indexed address,uint256)
```

Comment:
Still tracking the event?

Reply:
This has always been there to track pool share transfers. It uses the ERC20 contract as a template, not a data source